### ISSUE TITLE
Undefined errors when including columns that have no data

### ISSUE THREAD

**Issue by jeffnm** [HSG-Library/print-slip-report#17]
Thank you for this app. We've found it very helpful since we started using it. 

We just started encountering errors when downloading the report with columns that do not have data in them. When downloading the report with columns selected where any requests are missing data in that column, there are now undefined errors and the file fails to download. Previously these columns would download with a blank cell, just like the default print slip report. 

We've reproduced this with multiple fields. When all of the requests have data in the checked column(s), then the download functions normally. As soon as even one of the requests is missing data in a checked column, the download gives these errors. 

Here are screenshots in 3 different browsers (each has it's own error, but I'm pretty sure they are all the same issue). 

<img width="442" alt="Screenshot 2024-05-29 at 10 46 40 AM" src="https://github.com/HSG-Library/print-slip-report/assets/651304/2779aa65-2355-4c51-b9c7-98484fe8df04">
<p>Safari</p>

<img width="435" alt="Screenshot 2024-05-29 at 10 48 15 AM" src="https://github.com/HSG-Library/print-slip-report/assets/651304/f108e5b5-2521-4626-b48a-93414ccaf231">
<p>Chrome</p>

<img width="430" alt="Screenshot 2024-05-29 at 10 43 47 AM" src="https://github.com/HSG-Library/print-slip-report/assets/651304/af6fbf34-85be-44e2-8558-8268533fef1f">
<p>Firefox</p>

I'm happy to provide further information or testing as needed.

**Comment by d22** [comment]
Hi @jeffnm 
Thank you for your report. Can you please post the according console output? And also tell me what columns you have selected.
BR
Jonas

**Comment by jeffnm** [comment]
@d22 ,

Today, the download is working again! It must have been a temporary issue with Alma. We did test it on 3 different machines and accounts over 24 hours before reporting it, but we're thankful it's working again. 

To answer your questions in case it's still helpful, the console in Firefox was empty when I looked yesterday before submitting this issue. That is, I cleared the console, then clicked the Download button and nothing new appeared in the console, but the error appeared on screen. Since I can't reproduce the issue now, I can't try again or check other browser consoles. 

The fields that we saw causing the error included "Accession Number", "Volume", and even "Barcode" (we had a few requests that were missing barcodes). The simplest set that I tested that gave the error was "Title" and one of those 3 fields. 

I'm going to close this issue since it has cleared up on it's own. Thanks again!
